components/style/properties/longhand/font.mako.rs

```diff
@@ -511,44 +520,58 @@ ${helpers.single_keyword_system("font-variant-caps",
         #[derive(PartialEq, Eq, Copy, Clone, Hash, Debug)]
         #[cfg_attr(feature = "servo", derive(HeapSizeOf, Deserialize, Serialize))]
         #[repr(u16)]
-        pub enum T {
+        pub enum Weight {
             % for weight in range(100, 901, 100):
                 Weight${weight} = ${weight},
             % endfor
         }
+
+        #[derive(PartialEq, Eq, Copy, Clone, Debug)]
+        #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+        pub struct T {
```

Comment:
I... don't understand. Why do you want to change `font_weight::computed::T`? I don't think we want to change type of computed value of `font-weight` property. What we want to change is the value of `font-weight` descriptor in `@font-face` rule.

I think it is probably better just having a separate type like
```rust
enum FontWeight {
    Weight(font_weight::Weight),
    Normal,
    Bold,
}
```
for `font-weight` descriptor.